### ISSUE TITLE
Problem when using "PCA FLAVOUR: Symbols" under "Output" in the GUI

### ISSUE THREAD

**Issue by MatthiasSL** [computationalstylistics/stylo#52]
I am using stylo to do PCA on a corpus, and when I use the "Classic", "Loadings" and "Technical" PCA flavours, stylo generates a plot just fine, but when I try to generate one using the "Symbols" flavour, I get the following error (the first two words are just Danish for "error in"):

`
Fejl i fun(key = base::quote(list(columns = 2, text = list(NULL), points = list( : 
  first component of text must be vector of labels
`

I am analyzing a corpus consisting of english-language files with a few special characters here and there, but all unicode-based. Considering the fact that the other flavours works just fine, I'm assuming its not an issue with the corpus. When I first got the error, I noticed I was on a lower version of R (4.1.2) than the latest version of stylo was made for (4.1.3), so I updated to the newest version of R (4.2.0), but the error persists.

It appears a similar error has happened before (although that user was running everything via the command line): #43

**Comment by jmclawson** [comment]
What version of stylo are you using? I think it’s been updated in the latest versions.

**Comment by tillgrallert** [comment]
@jmclawson the error is very much present in the current version of stylo (0.7.4) as available from CRAN.  on macOS 10.14.5, R 4.0.5, I get:

```r
 Error in fun(key = base::quote(list(columns = 2, text = list(NULL), points = list( : 
  first component of text must be vector of labels 
```

**Comment by jmclawson** [comment]
You're right! I see it, too. It's a variation of a problem that was fixed in a previous update--but it turns out not to have been fixed completely. A fix is on the way.

(Details: The `data.frame()` command in previous versions of R defaulted to `stringsAsFactors = TRUE`, but this changed sometime in ~2020 to `stringsAsFactors = FALSE`. This bug showcases one of the moments where stylo hasn't yet corrected for it.)

**Comment by jmclawson** [comment]
On second thought... It seems my debugging time this evening was spent tracking down the exact same error I fixed in 2020. :upside_down_face: 

So the bug is fixed on GitHub but hasn't yet been propagated to CRAN. To install the GitHub version, use the `remotes` package to download and install it directly: `remotes::install_github("computationalstylistics/stylo")`.

**Comment by tillgrallert** [comment]
@jmclawson thank you very much for the quick response and fix. Exceptional!

**Comment by tillgrallert** [comment]
Another question though: I don't see any new commits to any of the branches in this repository.

**Comment by jmclawson** [comment]
The bug was fixed in my August 2020 pull request:[set stringsAsFactors=TRUE for COOR data.frame](https://github.com/computationalstylistics/stylo/commit/3783fa55cd0295d90c40cca3d33263cb1bedc8f0).

The GitHub version has had the fix incorporated since the request was merged in December 2021, but the CRAN version is older.

**Comment by computationalstylistics** [comment]
Dear Both, thanks a lot for a very fruitful discussion. I have indeed merged @jmclawson 's improvements some time ago, then I accidentally messed with the branches, and then pushed the stuff to CRAN without the improvements :-(   As far as I can see, it works fine in the current state. I'll finalize the ver. 0.7.5 in the upcoming days, and push it again to CRAN.
